en/docs/deploy/monitor/log4j-mask-sensitive-information-in-logs.md

```diff
@@ -0,0 +1,51 @@
+# Configure Log Masking with Log4j
+
+There can be business sensitive information that are added to logs in 
+the product console and/or Carbon log files. When these logs
+are analyzed, the information is exposed to those who check this.
+
+To avoid this potential security pitfall, users can mask sensitive
+information.
+
+To avoid this potential security pitfall, users can mask sensitive
+information from the log file at the time of logging. In this feature,
+you can define patterns that need to be masked from the logs. This is
+particularly useful in the cases of credit card numbers, access tokens,
+etc.
+
+To configure this feature, follow the instructions given below.
+
+---
+
+## Enable log masking
+
+1.  Log masking is enabled by default. However, you need to configure required masking patterns.
+
+2.  These patterns are configured in the `<IS-HOME>/repository/conf/wso2-log-masking.properties`. You can change 
```

Comment:
`wso2-log-masking.properties` is the internal file afaiu? Users won't have to access this for any reason, right? If so, we don't need to mention this in the doc.

Reply:
Users might want to get an insight on what patterns to add apart from the defaults.